01-builder-visualization/exercises/twitter-dogs.md

```diff
@@ -0,0 +1,202 @@
+# Twitter Pop-Ups & Time Slider Visualization
+
+* *Degree of Difficulty*: :star::star::star:
+* *Goal*: Create a time-enabled visualization that includes useful pop-ups using data pulled from Twitter with BUILDER.
+* *Features Highlighted*:
+  * Style:
+    - Enabling time-based elements of your data for advanced visualization;
+    - Making the most of your data by adding and altering pop-ups to display useful information at a glance.
+  * Layers: how to work with layer nodes
+  * Widgets: Formulas + Histograms
+* *Datasests needed*:
+  * `twitter_dogs` dataset: twitter data on based on dog breed search terms from 6/11/2017 - 7/10/17.
+
+
+1. [Import and create map](#map) <br>
+2. [Style](#sty) <br>
+3. [Widgets](#widgets) <br>
+4. [Publish and share map](#pub) <br>
+
+<hr>
+
+## 1. Import and create map <a name="map"></a>
+
+### 1. 1. Import `twitter_dogs` file into your DATASETS dashboard.
+
+* Download the dataset from [here](https://builder-demo.carto.com/api/v2/sql?q=SELECT+*+FROM+twitter_dogs&format=csv&filename=twitter_dogs).
+* Go to your account and import it using the **NEW DATASET** options.
+* Select **CONNECT DATASET**.
+* Drag and drop the file.
+
+### 1. 2. Dataset view
+
+* Take a look on the dataset.
```

Comment:
prob "on" should be "at", take a look on the dataset sounds awkward grammatically